### PR TITLE
STAC-14536 Describes limitation of S3 encrypted buckets for backups

### PR DESCRIPTION
**Pull request by vpartington**

### CODE REVIEW

File: setup/data-management/backup_restore/kubernetes_backup.md [hunk 1]
```diff
@@ -36,6 +36,13 @@ The built-in MinIO instance can be configured to store the backups in three loca
 
 ### Backup to AWS S3
 
+{% hint style="warning" %}
+The S3 buckets used to store the backups can be encrypted with an Amazon S3-managed key (SSE-S3). Encryption with AWS KMS keys stored in AWS Key Management Service (SSE-KMS) is not supported and will result in error such as this one in the Elasticsearch logs:
```
Comment (gggina) [inline setup/data-management/backup_restore/kubernetes_backup.md:40]: "The S3 buckets used to store the backups can be encrypted with an Amazon S3-managed key (SSE-S3)."

Does this mean that encryption can be used on S3 buckets user to store backups?

Comment (gggina) [inline setup/data-management/backup_restore/kubernetes_backup.md:40]: "Encryption with AWS KMS keys stored in AWS Key Management Service (SSE-KMS) is not supported"

Does this mean that encryption CANNOT be used on S3 buckets user to store backups?

File: setup/data-management/backup_restore/kubernetes_backup.md [hunk 2]
```diff
@@ -36,6 +36,13 @@ The built-in MinIO instance can be configured to store the backups in three loca
...
+```
+Caused by: org.elasticsearch.common.io.stream.NotSerializableExceptionWrapper: sdk_client_exception: Unable to verify integrity of data upload. Client calculated content hash (contentMD5: ZX4D/ZDUzZWRhNDUyZTI1MTc= in base 64) didn't match hash (etag: c75faa31280154027542f6530c9e543e in hex) calculated by Amazon S3.  You may need to delete the data stored in Amazon S3. (metadata.contentMD5: null, md5DigestStream: com.amazonaws.services.s3.internal.MD5DigestCalculatingInputStream@5481a656, bucketName: stackstate-elasticsearch-backup, key: tests-UG34QIV9s32tTzQWdPsZL/master.dat)",  
+```
```
Comment (gggina) [inline setup/data-management/backup_restore/kubernetes_backup.md:43]: What to do if you see this error in the ES logs?

File: setup/data-management/backup_restore/kubernetes_backup.md (hunk 1 above)
Comment (vpartington) [reply setup/data-management/backup_restore/kubernetes_backup.md:40]: Yes. But only with SSE-S3 keys, not with SSE-KMS.

**Pull request merged**